libraries/AP_Scripting/AP_Scripting.cpp

```diff
@@ -97,12 +97,13 @@ void AP_Scripting::init(void) {
     }
 
     const char *dir_name = SCRIPTING_DIRECTORY;
-
-    if (!AP::FS().mkdir(dir_name)) {
-        gcs().send_text(MAV_SEVERITY_INFO, "Lua: created new directory (%s)", dir_name);
-    } else if (errno != EEXIST) {
-        gcs().send_text(MAV_SEVERITY_INFO, "Lua: failed to create (%s)", dir_name);
-        return;
+    const bool directory_created = !AP::FS().mkdir(dir_name);
```

Comment:
Please remove this temp.  It's just polluting the namespace of the rest of the function.